CMakeLists.txt

```diff
@@ -94,15 +96,41 @@ if(NOT DAEMON_EXTERNAL_APP)
     # can be loaded by daemon with vm.[sc]game.type 2
     option(BUILD_GAME_NATIVE_EXE "Build native executable, which might be used for better performances by server owners" OFF)
 
-    if (CMAKE_SYSTEM_NAME STREQUAL CMAKE_HOST_SYSTEM_NAME)
+    # The NaCl SDK only runs on amd64 or i686.
+    if (CMAKE_SYSTEM_NAME STREQUAL CMAKE_HOST_SYSTEM_NAME
+    AND (ARCH STREQUAL "amd64" OR ARCH STREQUAL "i686"))
         # can be loaded by daemon with vm.[sc]game.type 0 or 1
-        option(BUILD_GAME_NACL "Build the NaCl gamelogic modules, required to host mods." OFF)
-        option(BUILD_GAME_NACL_NEXE "Build the NaCl \"nexe\" files from the \"pexe\" files." ON)
-        mark_as_advanced(BUILD_GAME_NACL_NEXE)
+        set(NACL_ALL_TARGETS "amd64;i686;armhf")
+        set(BUILD_GAME_NACL_TARGETS "none" CACHE STRING "Build the NaCl \"nexe\" files for those architectures, values: ${NACL_ALL_TARGETS}, or pseudo-targets like all, native or none.")
```

Comment:
I think you will probably need to update https://wiki.unvanquished.net/wiki/Making_an_awesome_mod after this is merged so that people know how to make VM files. And I'll have to update the testing server code so that it still builds VM files.